src/commands/restore.yml

```diff
@@ -0,0 +1,25 @@
+description: >
+  Restores the latest backup available of an environment to Skpr.
+
+parameters:
+  source:
+    type: string
+    default: ""
+    description: "The source environment to grab the backup from"
+  target:
+    type: string
+    default: ""
+    description: "The target environment to restore the backup to"
+  backup:
+    type: string
+    default: ""
+    description: "If applicable, the backup name to restore"
```

Comment:
What does "If applicable mean?". What happens if you don't provide it?

Reply:
I will correct the sentence - however it will choose the latest available backup from the source if left empty.

Reply:
I've updated the description to show what happens if left empty, vs what the parameter actually does.